docarray/document/mixins/_property.py

```diff
@@ -1,12 +1,12 @@
-# auto-generated from /Users/hanxiao/Documents/docarray/scripts/gen_doc_property_mixin.py
-from typing import TYPE_CHECKING, Dict, List, Optional
+# auto-generated from /Users/marco/code/opensource/docarray/scripts/gen_doc_property_mixin.py
+from typing import TYPE_CHECKING, Dict, List, Optional, Union, Any
 
-if TYPE_CHECKING:  # pragma: no cover
-    from docarray.score import NamedScore
-    from docarray.array.match import MatchArray
-    from docarray.array.chunk import ChunkArray
-    from docarray import DocumentArray
-    from docarray.typing import ArrayType, StructValueType, DocumentContentType
+if TYPE_CHECKING:
+    from ...score import NamedScore
```

Comment:
please keep the absolute imports

Reply:
I can do that: it is a simple change to the script `scripts/gen_doc_property_mixin.py`